controllers/dscinitialization/utils.go

```diff
@@ -440,3 +444,41 @@ func (r *DSCInitializationReconciler) createUserGroup(ctx context.Context, dscIn
 
 	return nil
 }
+
+func IsDefaultAudiences(specAudiences *[]string) bool {
```

Comment:
Not sure we need this exported... If only for tests, I think we can use `GetEffectiveClusterAudiences` as an entry point there.

Reply:
done 
e349c3c